File: JavaApplication73/src/Boletin13_1/Xogador.java
package Boletin13_1;

/**
 *
 * @author clamascabaleiro
 */
public class Xogador extends Seleccion{
    private int dorsal;
    private String demarcacion;

    public Xogador() {
    }

    public Xogador(int id, String nome, String apelido, int idade, int dorsal, String demarcacion) {
        super.setId(id);
        super.setNome(nome);
        super.setApelido(apelido);
        super.setIdade(idade);
        this.dorsal = dorsal;
        this.demarcacion = demarcacion;
    }

    public int getDorsal() {
        return dorsal;
    }

    public void setDorsal(int dorsal) {
        this.dorsal = dorsal;
    }

    public String getDemarcacion() {
        return demarcacion;
    }

    public void setDemarcacion(String demarcacion) {
        this.demarcacion = demarcacion;
    }
    
    @Override
    public void viaxar(){
        System.out.println("viaxan os xogadores");
    }
    
    public void xogarPartido(){
       System.out.println("o xogador " + super.getNome() + " xoga");
    }
    
    public void entrenar(){
       System.out.println("o xogador " + super.getNome() + " entrena");
    }

    @Override
    public String toString() {
        return super.toString() + ", Xogador{" + "dorsal=" + dorsal + ", demarcacion=" + demarcacion + '}';
    }

}

File: JavaApplication73/src/Boletin13_1/Xogador.kt
package Boletin13_1

/**
 *
 * @author clamascabaleiro
 */
//Constructor primario y constructor de la superclase Seleccion
class Xogador(var dorsal: Int , var demarcacion: String ,id: Int, nome: String, apelido: String, edade: Int) : Seleccion(id, nome, apelido, edade) {

    init{

    }

    @Override
    fun viaxar() {
        System.out.println("viaxan os xogadores")
    }

    fun xogarPartido() {
        System.out.println("o xogador " + super.getNome() + " xoga")
    }

    fun entrenar() {
        System.out.println("o xogador " + super.getNome() + " entrena")
    }

    @Override
    fun toString(): String {
        return super.toString() + ", Xogador{" + "dorsal=" + dorsal + ", demarcacion=" + demarcacion + '}'.toString()
    }

}
